superset/assets/src/components/TableSelector.jsx

```diff
@@ -127,6 +127,10 @@ export default class TableSelector extends React.PureComponent {
        return SupersetClient.get({ endpoint })
         .then(({ json }) => {
           const filterOptions = createFilterOptions({ options: json.options });
+          json.options.forEach((option) => {
+            // eslint-disable-next-line no-param-reassign
```

Comment:
Create a new objection from `options` and add the `title` fields instead to avoid the warning?

Reply:
True that, I updated the commit!